src/types/utils/whitelistedReportKeys.ts

```diff
@@ -63,6 +63,8 @@ type WhitelistedReport = OnyxCommon.OnyxValueWithOfflineFeedback<
         };
         // eslint-disable-next-line @typescript-eslint/naming-convention
         private_isArchived: unknown;
+        state: unknown;
+        welcomeMessage: unknown;
```

Comment:
I think this comment is strange. The whole thing is pretty confusing. It's just listing all the fields in the report and asserting that they're in the report type. I don't find any value in that because it's totally circular.

I found the [related issue](https://github.com/Expensify/App/issues/51867), and the goal there is to move client only keys out of the report type. So now I understand the purpose. It also doesn't really make sense to limit that to the OpenApp call, I think OpenReport is also a valid way to tell which keys come from the server. I checked the response and I see that welcomeMessage is returned but state is not, so I'll remove state.


Thanks for pointing this out.